packages/core/src/sass/add-or-manage.scss

```diff
@@ -36,10 +36,11 @@
         }
         &__deposit-warn {
             width: 70%;
-            margin: 2.8rem auto 0;
+            margin: 2.8rem auto 8rem;
             padding: 10px;
             background-color: var(--general-section-1);
             border-radius: $BORDER_RADIUS;
+            text-align: center;
```

Comment:
`Text` is our reusable component that we can give props to instead of adding these properties to classes. `Text` props include color, size (font size), weight (font weight), line_height and align. Please try to use `Text` and its props wherever you can.

Reply:
Done

Reply:
If you're using align in `Text`, this isn't needed here. :)

Reply:
thanks for this recommendation, will transfer align to the Text.